layouts/millennium_r4_sidebar.html

```diff
@@ -15,6 +15,11 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
           <li><a href="/millennium/r4/devices/device/">Device</a></li>
         </ul>
       </li>
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/millennium/r4/financial/">Financial</a></h3>
+        <ul class="js-guides">
+          <li><a href="/millennium/r4/financial/account/">Account</a></li>
+        </ul>
+      </li
```

Comment:
this renders incorrectly on the site, the html tags are not correct (and causes issues rendering Encounter)

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/827e1ebe11e8550a24530b7d74d2878100075ead#diff-1abdee660a30300c05753561e598f2dcR22

Reply:
that change didn't fix the issue. need an opening li tag at least, i haven't re-rendered to check what if anything else is needed

Reply:
fixed it this time. https://github.com/cerner/fhir.cerner.com/pull/291/commits/56fbe6fa5a7ad82d99a70b23bd2bdd5b3788615f#diff-1abdee660a30300c05753561e598f2dcR18